tabs/tab_survival.py

```
@@ -268,244 +241,484 @@ def surv_ui(namespace: str) -> ui.TagChild:
+                if matched:
+                    default_subgr = matched[0]
+                    break
+            
+            # Fallback: ถ้าไม่เจอ keyword เลย ให้เลือกคอลัมน์แรกสุด
+            if default_subgr == "Select..." and cols:
+                default_subgr = cols[0]
+                
+            # KM Curves
+            ui.update_select("surv_time", choices=numeric_cols, selected=default_time)
+            ui.update_select("surv_event", choices=cols, selected=default_event)
+            ui.update_select("surv_group", choices=["None"] + cols)
```

Ruff (0.14.10)

Consider `["None", *cols]` instead of concatenation

Replace with `["None", *cols]`

(RUF005)